tasks/pip.yml

```diff
@@ -180,69 +139,48 @@
   register: result
   until: result is success
   notify: Reload application
+  tags:
+    - molecule-idempotence-notest
 
 - name: Install psycopg2
   when: openwisp2_database.engine in ["django.db.backends.postgresql", "django.contrib.gis.db.backends.postgis"]
   pip:
-    name: psycopg2
+    name: psycopg2-binary
     state: latest
     virtualenv: "{{ virtualenv_path }}"
     virtualenv_python: "{{ openwisp2_python }}"
-  retries: 5
-  delay: 10
+  retries: 3
+  delay: 5
   register: result
   until: result is success
   notify: Reload application
 
-- name: Install MySQL-python
-  when: openwisp2_database.engine in ["django.db.backends.mysql", "django.contrib.gis.db.backends.mysql"]
+- name: Install MySQL-python for Python 2
+  when: 
+    - openwisp2_database.engine in ["django.db.backends.mysql", "django.contrib.gis.db.backends.mysql"]
+    - openwisp2_python is version('3.0', '<')
   pip:
     name: MySQL-python
     state: latest
     virtualenv: "{{ virtualenv_path }}"
     virtualenv_python: "{{ openwisp2_python }}"
-  retries: 5
-  delay: 10
-  register: result
-  until: result is success
-  notify: Reload application
-
-- name: Install raven (sentry client)
-  when: openwisp2_sentry.get('dsn')
-  pip:
-    name: raven
-    state: latest
-    virtualenv: "{{ virtualenv_path }}"
-    virtualenv_python: "{{ openwisp2_python }}"
-  retries: 5
-  delay: 10
+  retries: 3
+  delay: 5
   register: result
   until: result is success
   notify: Reload application
 
-- name: Install django-celery-email
+- name: Install mysqlclient for Python 3
+  when:
+    - openwisp2_database.engine in ["django.db.backends.mysql", "django.contrib.gis.db.backends.mysql"]
+    - openwisp2_python is version('3.0', '>=')
   pip:
-    name: django-celery-email
+    name: mysqlclient
     state: latest
     virtualenv: "{{ virtualenv_path }}"
     virtualenv_python: "{{ openwisp2_python }}"
-  when: openwisp2_email_backend == "djcelery_email.backends.CeleryEmailBackend"
```

Comment:
As I suggested in the previous review, don't add packages to requirements.txt which are installed conditionally.

Reply:
we could install it anyway, it shouldn't be a big deal, what do you think?